methods.md

```diff
@@ -141,13 +159,52 @@ Use this method to verify that the pin code entered by the end-user matches the
 <tr><th>Name</th><th>Data Type</th><th>Description</th><th>Mandatory</th></tr>
 <tr><td>ServiceCredentials</td><td>ServiceCredentials</td><td>See <a href="methods.md#servicecredentials">here</a> for details</td><td>Yes</td></tr>
 <tr><td>Msisdn</td><td>String</td><td>The mobile subscription number associated with the verification code.</td><td>Yes</td></tr>
-<tr><td>VerificationCode</td><td>string</td><td>The code to be verified.</td><td>Yes</td></tr>
+<tr><td>VerificationCode</td><td>String</td><td>The code to be verified.</td><td>Yes</td></tr>
 </table>
 
 ### Response types
 N/A.
 
 
+## PreAuthorize
+
+PreAuthorized payments are recurring payments where the End User have confirmed that the Pay transaction for a merchant can be executed without End User confirmation on each transaction.
+
+This method returns a token to be used in subsequent Pay requests. The token is issued by Strex and does not expire. However it can be manually invalidated by Strex in some situations if the Code of Conduct is violated.
+
+Please note that the End User needs to be a user on the Strex platform for this to be successful.
+
+NB This method is only available on the REST API
+
+
+### Request types
+<table>
+<tr><th>Name</th><th>Data Type</th><th>Description</th><th>Mandatory</th></tr>
+<tr><td>ServiceCredentials</td><td>ServiceCredentials</td><td>See <a href="methods.md#servicecredentials">here</a> for details</td><td>Yes</td></tr>
+<tr><td>Msisdn</td><td>String</td><td>The mobile subscription to generate a token for.</td><td>Yes</td></tr>
+<tr><td>TransactionId</td><td>String</td><td>Your identifier for the request - is returned in the response.</td><td>Yes</td></tr>
+<tr><td>Platform</td><td>Enum</td><td>Which Puzzel platform to validate ServiceCredentials against<br><br>
+0 = DirectPay <br>
+1 = Sms</td><td>Yes</td></tr>
+<tr><td>SecurityLevel</td><td>Enum</td><td>Which security level the authorization request should use.<br><br>
+1 = None <br>
+2 = Confirmation<br>
+3 = Pin (not in use)</td><td>Yes</td></tr>
+<tr><td>ConfirmationChannel</td><td>Enum</td><td>If security level is set to 'Confirmation', you can specify which channel the confirmation message should use.<br><br>
+1 = USSD <br>
+2 = SMS</td><td>No</td></tr>
+<tr><td>ShortNumber</td><td>String</td><td>If ConfirmationChannel is set to 'SMS', you can specify the originator prefix of the sms message here. Default is '2222' from Strex.</td><td>Yes</td></tr>
```

Comment:
If this is mandatory, why is default 2222? Should it be Mandatory = No, then if not set it defaults to 2222? Only short number is possible, not alphanumeric originator (like Puzzel) ?

Reply:
Typo, it's not mandatory. Added a line to emphasize this is a prefix as Strex adds 10 digits to it when sending the sms. So if you add something else than 4 digits you still get the sms, but won't be able to reply to it...